packages/demand-api/README.md

```diff
@@ -125,13 +126,42 @@ Fare service:
 ```
 const fareService = new FareService(httpService)
 
+```
+
 Payment service:
 
 ```
-
 const paymentService = new PaymentService(httpService)
 
-````
+```
+
+Flags service:
+
+```
+const flagsService = new FlagsService(httpService)
+
+```
+
+### Example
+
+Let's see how to use one of the services, listed above.
+
+For example, you need to get a list of quotes for trip from one location to another. For this purpose you have to use Quotes service and call quotesSearch method:
+
+const quotesSearchParams = {
+origin_place_id: "123456778qwertyu"
+destination_place_id: "0988765poiuyt"
+local_time_of_pickup?: "2020-05-12T10:00"
+}
+
+const quotesResponse =
+quotesService.quotesSearch(quotesSearchParams)
+.then(result => {
+//handleResult
+})
+.catch(error => {
+//handle error
+})
```

Comment:
<img width="967" alt="Screen Shot 2020-05-11 at 11 19 39" src="https://user-images.githubusercontent.com/6632944/81539737-58e5c300-9379-11ea-9dfa-5e922c9e9096.png">

Reply:
That's how it will look, is it what you want ? :)